### DISCUSSION TITLE
Waiting for first page view

### DISCUSSION THREAD

**Discussion by ashispavan** [plausible/analytics#1769] [Self-Hosted Support]
Hi,

I am self hosting a Plausible instance on Digital Ocean, and I have added the necessary script tags to my website. I am able to view the calls for script as well as api/event in the Networks tab, but the dashboard still shows this:
![plausible](https://user-images.githubusercontent.com/11900337/158451469-95f4ddbe-b2d0-4287-b272-8cb4d5a9598d.png)

Is there a way I can debug more and figure out where the issue is?

Any help is much appreciated, thanks!

**Comment by SaneChoice** [comment]
Did you manage to resolve this @ashispavan?

**Comment by denleschae** [comment]
I was stuck in the same spot when setting up Plausible on Digital Ocean. I walked through all of the troubleshooting steps Plausible had and everything was correct. In the end I got it all working by following this guide from Digital Ocean. I already had everything setup via Plausible's guide, but I picked up from the `docker-compose.yml` section setting up the localhost interface. Once I got their reverse proxy setup and and SSL certificate installed for my subdomain (via the guide) it all worked. Note that when you add the plausible script to your site, omit the 8000 port from the URL. No port required.

https://www.digitalocean.com/community/tutorials/how-to-install-plausible-analytics-on-ubuntu-20-04

**Reply by jforbush** [reply]
thanks for this post as I’ve been trying to figure out why data was coming through…then your comment of “omit the 8000 port from the URL” hit me like a rake in the head while walking across the yard!  doh!

**Comment by snaffi** [comment]
I've another problem. I don't see any HTTP requests in Network tab.
